packages/components/managed-data-table/src/data-table-settings/data-table-settings.js

```diff
@@ -0,0 +1,155 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import requiredIf from 'react-required-if';
+import SelectInput from '@commercetools-uikit/select-input';
+import Spacings from '@commercetools-uikit/spacings';
+import Constraints from '@commercetools-uikit/constraints';
+import DensityManager from './density-manager';
+import ColumnManager from './column-manager';
+import TABLE_SETTINGS_ACTIONS from '../constants';
+
+const COLUMN_MANAGER = 'columnManager';
+const DISPLAY_SETTINGS = 'displaySettings';
+
+const DataTableSettings = (props) => {
+  const [isColumnManagerOpen, setIsColumnManagerOpen] = useState(false);
+  const [isDisplayManagerOpen, setIsDisplayManagerOpen] = useState(false);
```

Comment:
If you see two useStates, use a useReducer. Maybe think about it :)

```
const useDisplaySettingsState = () => {
    const [ state, dispatch ] = useReducer();


    return { setColumnMnagerVisisble, setDisplayManagerOpen, isA, isB }
}
```

or but both `useStates` into a custom hook. Then the consumer doesn't care how the state is implemented all of a sudden :)